ccpp/data/GFS_typedefs.meta

```diff
@@ -4638,6 +4684,56 @@
   type = character
   kind = len=3
   active = (control_for_stochastic_land_surface_perturbation /= 0)
+[n_var_spp]
+  standard_name = number_of_spp_schemes_perturbed
+  long_name = number of spp schemes perturbed
+  units = count
+  dimensions = ()
+  type = integer
+[spp_prt_list]
+  standard_name =magnitude_of_perturbations_for_spp
+  long_name = magnitude of perturbations for spp
+  units = variable
+  dimensions = (number_of_spp_schemes_perturbed)
+  type = real
+  kind = kind_phys
+[spp_var_list]
+  standard_name = variables_to_be_perturbed_for_spp
```

Comment:
Why is this called "variables"? It seems like it is referring to a scheme type (pbl, rad, etc.). If so:

standard_name suggestion: perturbed_spp_schemes

Reply:
Updated.  This should indeed be schemes, not variables.  Thanks!